app/models/user.rb

```diff
@@ -2,16 +2,16 @@ class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :validatable
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] Good job adding `confirmable` but now when I register a new user I get the following error. To resolve this, kindly uncomment lines 25 to 28 in the devise migration file 👍 
- Error:
![Screenshot 2022-07-20 082138](https://user-images.githubusercontent.com/85113567/179922397-a8cfa8fc-520e-4ab9-95bb-873d1149ee28.png)